projects/dex-ui/src/components/Liquidity/QuoteDetails.tsx

```diff
@@ -197,22 +192,19 @@ const QuoteDetails = ({
 
     let priceDiff
     if (!newPrice || !oldPrice) {
-      priceDiff = TokenValue.ZERO;
+      return TokenValue.ZERO;
     } else if (newPrice?.eq(TokenValue.ZERO)) {
-      priceDiff = TokenValue.fromHuman(-100, 6);
+      return TokenValue.fromHuman(-100, 6);
     } else if (oldPrice?.eq(TokenValue.ZERO)) {
-      priceDiff = TokenValue.fromHuman(100, 6);
+      return TokenValue.fromHuman(100, 6);
     } else {
-      if (type === LIQUIDITY_OPERATION_TYPE.REMOVE) {
-        priceDiff = newPrice.div(oldPrice).mul(TokenValue.fromHuman(newPrice.gte(oldPrice) ? 100 : -100, 6));
-      } else {
-        priceDiff = newPrice.div(oldPrice).mul(TokenValue.fromHuman(100, 6)).sub(TokenValue.fromHuman(100, 6)).mul(TokenValue.fromHuman(-1, 6));
-      }
+      priceDiff = oldPrice.sub(newPrice).div(newPrice).mul(TokenValue.fromHuman(100, 6))
```

Comment:
Not clear what's going on here. Why are you multiplying to 100 with six digits?

Reply:
This is the price impact calculation, we multiply by 100 to get the percentage